.github/CONTRIBUTING.md

```diff
@@ -1,64 +1,65 @@
-[issue]: https://github.com/PlaceholderAPI/PlaceholderAPI/issues/new?template=change_request_placeholderapi.md
-[Discord]: https://helpch.at/discord
-[homepage]: http://contributor-covenant.org
-[version]: http://contributor-covenant.org/version/1/4/
+[issue]: https://github.com/PlaceholderAPI/PlaceholderAPI/issues/new
+[discord]: https://helpch.at/discord
+[Code of Conduct]: https://github.com/PlaceholderAPI/PlaceholderAPI/blob/master/CODE_OF_CONDUCT.md
+[wiki]: https://github.com/PlaceholderAPI/PlaceholderAPI/blob/master/wiki
 
-# Contributing
+# Contributing Guidelines
+We welcome everyone to contribute towards the PlaceholderAPI Project, but doing so will require you to follow specific rules to keep a consistent and welcoming way of contributing.
 
-When contributing to this repository, please first discuss the change you wish to make via [issue] or through [Discord] with the owners of this repository before making a change. 
+If you have any questions about this contributing file, feel free to contact us on our [Discord Server][discord].
 
-Please note we have a code of conduct, please follow it in all your interactions with the project.
+## Issues
+Like any other Project can you encounter bugs or a feature is missing for you in the plugin.  
+For that can you open an [issue] to report a bug or suggest a new feature to be added.
 
-## Pull Request Process
+When doing so, make sure you follow the below rules.
 
-1. Update the README.md with details of changes if it is affected by those changes.
-2. Try to prevent any breaking changes affecting the end-user (Developers using the API).  
-Mention any breaking changes that could affect the end-user.
-3. Document any public methods that the end-user might use.
+### Follow the Template
+We have Issue templates to help us get the required information more easly. Please follow the provided template when either filing a bug report or feature request.  
+We reserve the right to close your issue without warning for not following the templates.
```

Comment:
Not a fan of the word "right" in this context. Perhaps consider the following:
```suggestion
Your issue may be closed without warning for not following the template.
```

Although, I'd actually say that issues may be closed for any reason, not necessarily related to the template.